@vates/types/src/xo.mts

```diff
@@ -50,7 +50,7 @@ type BaseXoVm = BaseXapiXo & {
   blockedOperations: Record<VM_OPERATIONS, string>
   boot: Record<string, string>
   coresPerSocket?: number
-  cpuCap?: number
+  cpuCap?: number | null
```

Comment:
it seems that it comes from this code 
```js
tmp = obj.VCPUs_params
 tmp.cap && (vm.cpuCap = +tmp.cap)
 ```
 I am not sure to understand how vm.cpuCap can be something other than `number? ` ?

Reply:
`VCPUs_params.cap`is not necessarily a number.
E.g. `(172.16.210.11)`
```
[03:50 r620-l1 ~]# xe vm-param-get param-name=VCPUs-params uuid=b9f06a61-cbe6-4fc9-60dc-cb220d8645e8
cap: null
```

Reply:
You're right. But `+'null'` return `NaN` (so `null` when JSON parsed).

I fixed `xapi-object-..` to not return `NaN` as value

Reply:
TIL what + null means in JS 
nice catch